src/mrpro/operators/functionals/MSE.py

```diff
@@ -0,0 +1,50 @@
+"""MSE-Functional."""
+
+from collections.abc import Sequence
+
+import torch
+
+from mrpro.operators.functionals.L2NormSquared import L2NormSquared
+
+
+class MSE(L2NormSquared):
+    r"""Functional class for the mean square error.
+
+    This makes use of the functional L2NormSquared.
+    """
+
+    def __init__(
+        self,
+        weight: torch.Tensor | complex = 1.0,
+        target: torch.Tensor | None | complex = None,
+        dim: int | Sequence[int] | None = None,
+        divide_by_n: bool = True,
+        keepdim: bool = False,
+    ) -> None:
+        r"""Initialize a Functional.
+
+        We assume that functionals are given in the form
```

Comment:
Replace with
The MSE functional is given by 
:math:`f: C^N -> [0, \infty), x -> \| W (x-b)\|_2^2`,
    where :math:`W` is either a scalar or tensor that corresponds to a (block-) diagonal operator
    that is applied to the input. For more details also see :class:`mrpro.operators.functionals.L2NormSquared`

Reply:
1/N should maybe be mentioned in the eqoation already